subgraphs/compound-forks/src/mapping.ts

```diff
@@ -1877,7 +1877,8 @@ export function getOrCreateMarketHourlySnapshot(
   return snapshot;
 }
 
-function getTokenPriceUSD(
+// Return price in either USD or ETH, depending on oracle each fork uses
+function getTokenPrice(
```

Comment:
shouldn't this still be `getTokenPriceUSD()`? since the CallResult you are passing down is normalized to USD no matter what..

Reply:
ah good catch!